<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite ignored="0" hostname="Qualitrix-LinuxPC" failures="1" tests="1" name="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm" time="54.813" errors="0" timestamp="10 Dec 2018 03:46:53 GMT" skipped="0">
  <testcase classname="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm" name="verifyThatTopicsAreDisplayedOnMetaDataForm" time="54.813">
    <failure message="expected [Topic
Topics
Selected Topics] but found [Topic]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Topic
Topics
Selected Topics] but found [Topic]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at org.pageobjects.TopicTreePageObject.createBook_verifyTopicTreePopup(TopicTreePageObject.java:107)
	at org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm.verifyThatTopicsAreDisplayedOnMetaDataForm(VerifyThatTopicsAreDisplayedOnMetaDataForm.java:41)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
=======
<<<<<<< HEAD
<testsuite name="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm" tests="1" time="0.001" ignored="0" failures="0" timestamp="07 Dec 2018 09:32:40 GMT" skipped="1" hostname="QUALITRIX-PC" errors="0">
  <testcase name="verifyThatTopicsAreDisplayedOnMetaDataForm" time="0.001" classname="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm">
=======
<testsuite ignored="0" hostname="Qualitrix-LinuxPC" failures="0" tests="1" name="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm" time="0.000" errors="0" timestamp="07 Dec 2018 12:46:58 GMT" skipped="1">
  <testcase classname="org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm" name="verifyThatTopicsAreDisplayedOnMetaDataForm" time="0.000">
>>>>>>> fd04c9623bd7c1c59433d937381798497fdf2ad8
    <skipped/>
>>>>>>> 6565901281ae9a3a962beb76e038941897e5ac3b
  </testcase> <!-- verifyThatTopicsAreDisplayedOnMetaDataForm -->
</testsuite> <!-- org.testscript.VerifyThatTopicsAreDisplayedOnMetaDataForm -->
